agent/src/beerocks/slave/1905_beacon_query_to_vs.cpp

```diff
@@ -0,0 +1,113 @@
+
+#include "1905_beacon_query_to_vs.h"
+#include <bcl/beerocks_defines.h>
+#include <mapf/common/utils.h>
+
+namespace gate {
```

Comment:
:-)
There was a naming discussion before the decision was taken.
The bottom line of the discussion was "whatever name is selected, a comment will be"
So here is the logic, and the options, and personally I have no strong feeling about any of the options:
The functionality of this namespace is to convert data type from one domain to its counterpart in another domain. In this example 1905 beacon request to intel's counterpart (vendor specific)
Normally I would call this namespace `bridge` because it bridges between two worlds, two domains. In the large view of this concept it is not just translates from one domain to another, it also might take aggregation operations, filtering, etc (imagine that you need 2 1905 messages before you can construct a VS specific message. The `bridge`, in between the two domains, is the place to make it. )
This is the dependency structure: 1905 world (knows noting about vs) --> bridge (knows both worlds) --> vs specific (knows nothing about 1905)